recipes/ckanbatch-deploy.rb

```diff
@@ -60,16 +55,11 @@
 # Create job worker config files.
 #
 
-cookbook_file "/etc/supervisor/conf.d/supervisor-ckan-worker.conf" do
+cookbook_file "/etc/supervisord.d/supervisor-ckan-worker.ini" do
```

Comment:
would this cause issue for old AL1 ami's?

Reply:
It's running successfully on Amazon Linux 1 in DEV at the moment. The changes in `ckan-setup.rb` include setting up /etc/supervisord.d, and all the config is now being copied there.

Reply:
👍